README.md

```diff
@@ -31,3 +31,20 @@ match client.request(req).await {
     }
 }
 ```
+
+The above code will read your mauth configuration from a file in `~/.mauth_config.yml` which format is:
+```yaml
+common: &common
+  mauth_baseurl: https://<URL of MAUTH SERVER>
+  mauth_api_version: v1
+  app_uuid: <YOUR APP UUID HERE>
+  private_key_file: <PATH TO MAUTH KEY>
+  timeout: 100
+  open_timeout: 100
```

Comment:
What are these timeout configs doing?

Also, you might need to add configs related to v2:
https://github.com/mdsol/mauth-client-rust/blob/73598752addae00f824d4792a4d3dc2d8392f698/src/lib.rs#L92-L93